src/nls/it/strings.js

```diff
@@ -607,6 +606,7 @@ define({
     "HEALTH_DATA_DO_TRACK"                      : "Sì, mi piacerebbe condividere informazioni su come uso Brackets.",
     "HEALTH_DATA_NOTIFICATION_MESSAGE"          : "Al fine di migliorare Brackets stiamo implementando un nuovo Health Report che invia <strong>anonimo</strong> dati a Adobe su come usi Brackets. Questo rapporto aiuterà il gruppo di sviluppo e gli sviluppatori di estensioni sulle caratteristiche prioritarie, bug, problemi di usabilità e reperibilità.<br><br>È possibile visualizzare i dati inviati così come cambiare la vostra preferenza in qualsiasi momento visitando <strong>Help->Health Data Report</strong>. Ulteriori informazioni su Brackets Health Report e sullo stato <a href='https://github.com/adobe/brackets/wiki/Health-Data'>nella pagina wiki</a>.",
     "HEALTH_DATA_PREVIEW"                       : "Anteprima Health Data",
+    "HEALTH_DATA_PREVIEW_INTRO"                 : "<p>Al fine di migliorare {APP_NAME}, periodicamente invieremo limitate, statistiche <strong>anonime</strong> a Adobe su come usi {APP_NAME}. Queste informazioni aiutano priorità caratteristiche , trovare i bug , e individuare problemi di usabilità. <a href='https://github.com/adobe/brackets/wiki/Health-Data'>Per saperne di più {APP_NAME} Health Report</a> e come si avvantaggia la comunità di {APP_NAME} mentre la tua privacy é protetta.</p><p>Qui di seguito è una anteprima dei dati che saranno inviati nella tua prossima Health Report <em>se</em> é abilitato.</p>",
```

Comment:
1) No comma after `limitate` and `trovare i bug`
2) No space before commas

`<p>Al fine di migliorare {APP_NAME}, periodicamente invieremo limitate statistiche <strong>anonime</strong> a Adobe su come usi {APP_NAME}. Queste informazioni aiutano `**`ad assegnare una priorità ad alcune`**` caratteristiche, `**`a`**` trovare i bug e `**`a`**` individuare problemi di usabilità. <a href='https://github.com/adobe/brackets/wiki/Health-Data'>`**`Clicca qui `**`per saperne di più su {APP_NAME} Health Report</a> e su come si avvantaggia la comunità di {APP_NAME} `**`mantenendo`**` la tua privacy protetta.</p><p>`**`Ecco un'`**`anteprima dei dati che saranno inviati nella tua prossima Health Report `**`(`**`<em>se</em> abilitato`**`):`**`</p>`